database_schemas/cassandra/benchflow.cql

```diff
@@ -233,6 +234,32 @@ CREATE TABLE exp_cpu (
   PRIMARY KEY (experiment_id, container_id)
 );
 
+CREATE TABLE exp_cpu_core (
```

Comment:
You can think about having a list of value for each field. At the end we are always interested to look at these data all at once, so an ordered list of data points should be suitable.